src/Microsoft.DotNet.ProjectModel/Workspace/WorkspaceContext.cs

```diff
@@ -0,0 +1,426 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.ProjectModel;
+using Microsoft.Extensions.ProjectModel.Graph;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectModel.Workspace
```

Comment:
This is now the only class in this namespace, so we should remove the namespace.